viewer/jsdoc/returns.hbs

```diff
@@ -0,0 +1,13 @@
+{{#if returns}}
+{{#if returns.[0].description~}}
```

Comment:
returns.[0] ?

Reply:
Something specific with the hbs stuff. I don't know the nuances. The only reason I've added this file is to override the default. I wanted to add a newline to each different return value, which the default puts on a single line